pkg/placement/placement_manager.go

```diff
@@ -67,9 +100,11 @@ func NewPlacementManager(id string, kConfig *rest.Config) (*PlacementManager, er
 	podListerInformer := sharedInformers.Core().V1().Pods()
 	podInformer := podListerInformer.Informer()
 
+	defaultNodes := DiscoverNodes()
```

Comment:
```suggestion
	nodes := DiscoverNodes()
```

nodes are no longer hard-coded, so `defaultNodes` may not be a proper name.